scripts/commands/checkLinks.ts

```diff
@@ -295,4 +295,32 @@ async function determineHistoricalFileBatches(
   return result;
 }
 
+async function determineQiskitLegacyReleaseNotes(): Promise<FileBatch> {
+  const result: FileBatch[] = [];
+
+  const legacyVersions = (
+    await globby("docs/api/qiskit/release-notes/[!index]*")
+  )
+    .map((releaseNotesPath) =>
+      releaseNotesPath.split("/").pop()!.split(".").slice(0, -1).join("."),
+    )
+    .filter((version) => compareReleaseNotes(version, "0.45") < 0);
+
+  const legacyVersionsSorted = legacyVersions.sort((a, b) =>
+    compareReleaseNotes(a, b),
+  );
+
+  const toCheck = legacyVersionsSorted.map(
+    (legacyVersion) => `docs/api/qiskit/release-notes/${legacyVersion}.md`,
+  );
+
+  console.log(legacyVersionsSorted);
+
+  return await FileBatch.fromGlobs(
+    toCheck,
+    [`docs/api/qiskit/0.45/*`],
```

Comment:
Does this pass for current code? I think you will want to move the code you currently have marked as TODO to instead apply here.

Reply:
Yes, good catch! I have added it there and I will remove it in my other [PR](https://github.com/Qiskit/documentation/pull/865) :+1: